app/packs/controllers/dropzone_controller.js

```diff
@@ -35,6 +35,16 @@ export default class extends Controller {
     }
   }
 
+  checkForDuplicates(fileName) {
+    // Current files in dropZone
+    const fileNames = this.dropZone.files.map(file => { return file.name })
+    // Remove current fileName
+    fileNames.pop(fileName)
```

Comment:
Does pop take arguments?  https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/pop

Reply:
Fixed in latest commit; using `Array.splice`